src/functionalTest/resources/features/F-008/S-071.td.json

```diff
@@ -135,7 +134,7 @@
             "primaryLocation" : "[[ANYTHING_PRESENT]]",
             "jurisdiction" : "PRIVATELAW",
             "region" : "[[ANYTHING_PRESENT]]",
-            "workTypes" : "routine_work"
+            "workTypes" : "[[ANYTHING_PRESENT]]"
```

Comment:
we have to add proper worktypes. in consist values only we have to use ANYTHING_PRESENT. So user proper worktype here. and same for remaining places as well